internal/engine/filter/convert.go

```diff
@@ -3,6 +3,7 @@ package filter
 import (
```

Comment:
Please add a test for when double quoting fails here: https://github.com/mbrt/gmailctl/blob/master/internal/engine/filter/convert_test.go.